.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
+/node_modules
```

Comment:
Also, add a screenshot of your test coverage

Reply:
<img width="1435" alt="Screenshot 2019-05-15 at 16 30 38" src="https://user-images.githubusercontent.com/44126670/57790028-e2948d80-7731-11e9-8586-e232140a8601.png">
Test coverage